src/nat/retriever/sql_retriever/vanna_util.py

```
@@ -0,0 +1,948 @@
+    logger.info("=== Starting Vanna initialization ===")
+
+    # Get and train DDL from sqlite_master (if connected to SQLite)
+    if hasattr(vn, "run_sql") and vn.run_sql_is_set:
+        logger.info("Loading DDL from database...")
+        try:
+            # Try SQLite-specific query first
+            try:
+                df_ddl = vn.run_sql(
+                    "SELECT type, sql FROM sqlite_master WHERE sql is not null"
+                )
+            except Exception:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)